### ISSUE TITLE
Importing historical data from one device to another

### ISSUE THREAD

**Issue by hmonteiro** [Freeyourgadget/Gadgetbridge#1466]
I was having some connection problems with my band, so I decided to reset it and try to connect again. 
For some reason, it's Bluetooth address changed, so when I added it again to Gadgetbridge, it created a second device.
It seems to be working now, but I would like to move the history data from the old entry to the new one so I can have it in a single place.

Is there a way to do this?

#### Your wearable device is:

Mi Band 3

#### Your android version is:
7.1
#### Your Gadgetbridge version is:
0.32.4

**Comment by vanous** [comment]
Yes, this merge is possible and i did this. Be very careful with your data and make backups before you do this: https://codeberg.org/Freeyourgadget/Gadgetbridge/wiki/Data-Export-Import-Merging-Processing#merge-data-from-old-gadgetbridge-export
